satviz-parsers/src/main/java/edu/kit/satviz/parsers/ParsingException.java

```diff
@@ -1,4 +1,18 @@
 package edu.kit.satviz.parsers;
 
+/**
+ * This exception is thrown, whenever the file that's currently parsed is either
+ * syntactically incorrect or contains invalid data.
+ */
 public class ParsingException extends RuntimeException {
```

Comment:
Das sollte meiner Meinung nach eine checked Exception sein, die auch in den Methoden explizit deklariert wird.

Reply:
Nicht möglich, da man sonst die Iterator API bricht